pilot/pkg/security/authz/builder/testdata/http/custom-grpc-provider-no-namespace-out2.yaml

```diff
@@ -11,7 +11,7 @@ typedConfig:
         prefix: istio-ext-authz
   grpcService:
     envoyGrpc:
-      authority: outbound_.9000_._.my-custom-ext-authz.foo.svc.cluster.local
+      authority: my-custom-ext-authz.foo.svc.cluster.local
```

Comment:
Doesn't it matter if its mTLS or not?

For mTLS we need the old name. For non-mtls this name is fine

Reply:
Oh nevermind that is for SNI, this is authority